src/test/kotlin/step2/calculator/OperatorTest.kt

```diff
@@ -12,29 +12,39 @@ import step2.calculator.Operator.Companion.INVALID_OPERATOR_ERROR_MESSAGE
 import java.util.stream.Stream
 
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
-@DisplayName("계산을 담당하는 Operator 테스트")
+@DisplayName("계산을 담당하는 연산자 객체인 Operator 테스트")
 internal class OperatorTest {
-    @DisplayName("주어진 연산자가 올바른 경우 연산자 매핑 테스트")
+    @DisplayName("주어진 연산자가 올바른 경우 getOperator 함수를 호출하면 올바른 Operator 반환")
     @ParameterizedTest
-    @MethodSource("validOperator")
-    fun getOperatorWhenGivenValidOperator(givenOperator: String, expectedOperator: Operator) {
+    @MethodSource("correctOperatorValues")
+    fun given_CorrectOperatorValue_when_GetOperator_then_ReturnCorrectOperator(
+        givenOperator: String,
+        expectedOperator: Operator
+    ) {
         assertThat(Operator.getOperator(givenOperator)).isEqualTo(expectedOperator)
     }
 
-    @DisplayName("주어진 연산자가 올바르지 않은 경우 연산자 매핑 테스트")
+    @DisplayName("주어진 연산자가 올바르지 않은 경우 getOperator 함수를 호출하면 IllegalArgumentException 예외 발생")
```

Comment:
```suggestion
    @DisplayName("주어진 연산자가 올바르지 않은 경우 IllegalArgumentException 예외 발생")
```
getOperator 함수를 호출하면 이라는 문구는 없어도 될 것 같아요

Reply:
https://github.com/next-step/kotlin-racingcar/pull/361#discussion_r744846517